src/components/application_manager/include/application_manager/commands/command_request_impl.h

```diff
@@ -256,7 +283,7 @@ class CommandRequestImpl : public CommandImpl,
    * interface that returns response.
    * @return resulting code for sending to mobile application.
    */
-  mobile_apis::Result::eType PrepareResultCodeForResponse(
+  virtual mobile_apis::Result::eType PrepareResultCodeForResponse(
```

Comment:
@AKalinich-Luxoft Please rename parameters like you did in `IsResultCodeUnsupported(const ResponseInfo& first_iface_response...`